components/Premium/PremiumFeatures/PremiumFeatures.tsx

```diff
@@ -22,62 +25,114 @@ function PremiumFeatures() {
                 <thead>
                     <tr>
                         <th className={styles.featureColumnHeading}>Feature</th>
-                        <th>Free</th>
-                        <th>Premium</th>
+                        <th className={styles.premiumProductHeading}>Free</th>
+                        <th className={styles.premiumProductHeading}>Starter</th>
+                        <th className={styles.premiumProductHeading}>Premium</th>
+                        <th className={styles.premiumProductHeading}>Premium+</th>
                     </tr>
                 </thead>
                 <tbody>
                     <tr>
```

Comment:
please add chat colors (gray, white, green, gold)   
and support the project (❌,✅,✅,✅)  
bazaar data history (✅,✅,✅,✅)